native/src/main/java/bblfsh/bash/BashParser.java

```diff
@@ -0,0 +1,60 @@
+package bblfsh.bash;
+
+import com.ansorgit.plugins.bash.lang.BashVersion;
+import com.ansorgit.plugins.bash.lang.lexer.BashElementType;
+import com.ansorgit.plugins.bash.lang.lexer.BashLexer;
+import com.ansorgit.plugins.bash.lang.parser.BashParserDefinition;
+import com.ansorgit.plugins.bash.lang.parser.BashPsiBuilder;
+import com.ansorgit.plugins.bash.lang.parser.FileParsing;
+import com.intellij.core.CoreApplicationEnvironment;
+import com.intellij.core.CoreProjectEnvironment;
+import com.intellij.lang.ASTNode;
+import com.intellij.lang.ParserDefinition;
+import com.intellij.lang.PsiBuilder;
+import com.intellij.lang.impl.PsiBuilderFactoryImpl;
+import com.intellij.mock.MockProject;
+import com.intellij.openapi.Disposable;
+import com.intellij.psi.tree.IElementType;
+import com.intellij.openapi.extensions.ExtensionsArea;
+import com.intellij.openapi.extensions.Extensions;
+import com.intellij.lang.MetaLanguage;
+
+public class BashParser {
+    public static ASTNode run(final CharSequence code, final BashVersion version) {
+        final MockProject project = project();
+        final com.ansorgit.plugins.bash.lang.parser.BashParser parser =
```

Comment:
You are both right.  I'll change the name of my class from `BashParser` to `Parser`, since it is pretty clear from it package name (`bblfsh.bash`) what it is parsing.